### DISCUSSION TITLE
Stuck in Phase 3: Can't connect to robot via SSH

### DISCUSSION THREAD

**Discussion by GryhavenRest** [Hypfer/Valetudo#2144] [Q&A / Support]
### Are you already using Valetudo?

No, but in the process of starting it

### Your questions

Dear all,

I just started with the preparations for Valetudo on my Dreame L10s Ultra using the well written guide on the Valetudo home page.
I exectued Phase 1 and Phase 2 of the guide with success (I guess) and the device should now be in a rooted state.

When starting Phase 3 to install Valetudo I'm having issues to connect to the deivce via SSH. In the guide it says that I have to use the command _ssh -i ./your/keyfile root@192.168.5.1_ in the terminal. I'm a Linux Newbie so maybe thats the problem but i can't get the connection to work with that command. Since the description of this part in the guide is quite short so I guess this normally shouldn't be a critical step.

I open a new terminal and type in the command _ssh -i /home/cs/j6740a6065e9ac.id_rsa root@192.168.5.1_ but it never works. The terminal states the following:

_s@debian:~$ ssh -i /home/cs/j6740a6065e9ac.id_rsa root@192.168.5.1
ssh: connect to host 192.168.5.1 port 22: Connection refused_

I've been looking through the existing discussions but no one seems to have this problem or maybe I am to dumb to search correctly. I've tried it with zip file of the keys and with the .ppk key but nothing works. Why is it refusing the connection here?

Thank you very much for your help.

Best regards

**Comment by vivia** [comment]
First of all, to rule out network issues, make sure you're connected to the robot's access point and make sure you don't have a firewall running.

If you're sure it's not a network issue, try fully charging your robot and redo Phase 2 from the beginning.

**Reply by GryhavenRest** [reply]
Thanks for your respone!

I have turned the firewalls off. And I guess it's an important information than I am using VM Ware as a virtual machine. I changed the network connection to 'Bridged: Conntected directly to the physical network' and 'Replicate physical network connection state' but it still doesn't work. And also now it's not even refusing the connection, it just says 'Network is unreachable'.

How can I make sure if I am connected to the robot's access point? 

Is my command `ssh -i /home/cs/j6740a6065e9ac.id_rsa root@192.168.5.1` correct?

What do I have to look for regarding the state of my Dreame? I just turn it on and the wifi inidicator is flashing. I guess that's the way it should be.

**Reply by vivia** [reply]
If it says "network is unreachable" it's definitely a network error.

EDIT: I also just remembered that the docs explicitly say to not use a VM for this:

> Please use a native install for this, as VMs will usually be troublesome.

I suggest you ditch the VM idea for this and just boot from a Debian Live USB. You will need either a laptop or a desktop with a WiFi adaptor. When you select which WiFi network your computer should connect to, you will be able to see the access point of the Dreame robot. If you can't, long-press the two outer buttons until it spawns its AP again.
